Fix gh-2212: Migrate Scratch 1.4 download page

sjgllghr opened the pull request:
### Resolves:
Resolves #2212 

### Changes:
Migrates the Scratch 1.4 download load page to match new page style (following the example of the offline editor page)

### Test Coverage:
Ran unit tests, checked visual output and clicked all the links.

<img width="337" alt="screen shot 2018-10-30 at 12 10 43 am" src="https://user-images.githubusercontent.com/5944933/47701816-54631a80-dbd8-11e8-86b2-f4a5062edd29.png">

apple502j commented:
As a Japanese translator, it's a problem that the current Scratch 1.4 contains **wrong** translation. (For example, () mod () block)
Scratchr2 provides a way to solve this, but this one doesn't.
I am now asking other Japanese translators about this problem.

thisandagain commented:
@picklesrus Mind taking a look at this?

thisandagain commented:
Adding @chrisgarrity as a "pair" reviewer.

apple502j commented on src/views/scratch_1.4/scratch_1.4.jsx:
```diff
@@ -121,7 +127,28 @@ const OnePointFour = () => (
+                            id="onePointFour.resourcesA"
+                            values={{
+                                gettingStartedGuide: (
+                                    <a href="http://download.scratch.mit.edu/ScratchGettingStartedv14.pdf">
```
I think there is a localized version of this

chrisgarrity replied:
@apple502j I think that people have translated the 1.4 guide in the past, but it's never been systematically collected and shared on the Scratch website. So I think that's outside the scope for this PR, and probably not a priority right now.

picklesrus commented:
@sjgllghr Thank you for taking this on! Sorry I haven't gotten a chance to look at this yet!  My plan is to take a look at this by the end of the week. I'm new to scratch-www so @chrisgarrity  will follow up and make sure I don't miss anything and make sure any comments I may have are sending you in the right direction.  Thanks!

sjgllghr commented:
@picklesrus thanks for the review! I think I addressed all of the issues you pointed out. I’m not entirely sure I am handling the placeholders for translation correctly still. (e.g. I have placeholders around the “or” for linux install/download, but the placeholders are still separate translation entries..?)

picklesrus commented:
@sjgllghr I think that's right! It should give translators enough context to translate it now since they will know which text goes on either side of the "or".  

This PR looks good to me now. Since this is my first scratch-www review, I'm going to pass this off to @chrisgarrity to double check I didn't miss anything.  Thank you!

chrisgarrity commented on src/views/scratch_1.4/scratch_1.4.jsx:
```diff
@@ -0,0 +1,183 @@
+                            <p><FormattedMessage id="onePointFour.linuxBody" /></p>
+                            <ul className="installation-downloads">
+                                <li className="installation-downloads-item">
+                                    <FormattedMessage
```
Looks good!
